extension-manifest-v3/src/background/adblocker.js

```diff
@@ -349,7 +349,10 @@ ${scripts.join('\n\n')}}
   chrome.scripting.executeScript(
     {
       injectImmediately: true,
-      world: __PLATFORM__ === 'firefox' ? undefined : 'MAIN',
+      world:
+        chrome.scripting.ExecutionWorld?.MAIN ?? __PLATFORM__ === 'firefox'
```

Comment:
Are you sure about the priority of `??` against the `?` From the code styling, it looks like when L353 is truth then the result is `undefined`. If this is not your intention, I would add brackets from `__PLATFORM__ === 'firefox`...

I checked, and definitely it's not the same:

```js
 chrome.scripting.ExecutionWorld?.MAIN ??
        (__PLATFORM__ === 'firefox' ? undefined : 'MAIN'),
```

Reply:
good catch

Reply:
I actually didn't know about the difference in priority of `||` and `??`